eng/Net9Workarounds.targets

```diff
@@ -0,0 +1,137 @@
+<Project>
+  <PropertyGroup>
+    <!-- Workaround while there is no SDK available that understands the TFM; suppress unsupported version errors. -->
+    <NETCoreAppMaximumVersion>99.9</NETCoreAppMaximumVersion>
+
+    <UpdateAspNetCoreKnownFramework
+      Condition=" '$(UpdateAspNetCoreKnownFramework)' == '' ">true</UpdateAspNetCoreKnownFramework>
+  </PropertyGroup>
+
+  <!-- Workaround when there is no vNext SDK available; copy known apphost/framework reference info from 8.0. -->
+  <ItemGroup>
+    <KnownAppHostPack
+      Include="@(KnownAppHostPack->WithMetadataValue('TargetFramework', 'net8.0'))"
+      TargetFramework="$(LatestTargetFramework)"
+      Condition="'@(KnownAppHostPack->Count())' != '0' AND
+        !(@(KnownAppHostPack->AnyHaveMetadataValue('TargetFramework', '$(LatestTargetFramework)')))" />
+    <KnownRuntimePack
+      Include="@(KnownRuntimePack->WithMetadataValue('TargetFramework', 'net8.0'))"
+      TargetFramework="$(LatestTargetFramework)"
+      Condition="'@(KnownRuntimePack->Count())' != '0' AND
+        !(@(KnownRuntimePack->AnyHaveMetadataValue('TargetFramework', '$(LatestTargetFramework)')))" />
+    <KnownFrameworkReference
+      Include="@(KnownFrameworkReference->WithMetadataValue('TargetFramework', 'net8.0'))"
+      TargetFramework="$(LatestTargetFramework)"
+      Condition="'@(KnownFrameworkReference->Count())' != '0' AND
+        !(@(KnownFrameworkReference->AnyHaveMetadataValue('TargetFramework', '$(LatestTargetFramework)')))" />
+    <KnownILLinkPack
+      Include="@(KnownILLinkPack->WithMetadataValue('TargetFramework', 'net8.0'))"
+      TargetFramework="$(LatestTargetFramework)"
+      Condition="'@(KnownILLinkPack->Count())' != '0' AND
+        !(@(KnownILLinkPack->AnyHaveMetadataValue('TargetFramework', '$(LatestTargetFramework)')))" />
+    <KnownILCompilerPack
+      Include="@(KnownILCompilerPack->WithMetadataValue('TargetFramework', 'net8.0'))"
+      TargetFramework="$(LatestTargetFramework)"
+      Condition="'@(KnownILCompilerPack->Count())' != '0' AND
+        !(@(KnownILCompilerPack->AnyHaveMetadataValue('TargetFramework', '$(LatestTargetFramework)')))" />
+    <KnownWebAssemblySdkPack
+      Include="@(KnownWebAssemblySdkPack->WithMetadataValue('TargetFramework', 'net8.0'))"
+      TargetFramework="$(LatestTargetFramework)"
+      Condition="'@(KnownWebAssemblySdkPack->Count())' != '0' AND
+        !(@(KnownWebAssemblySdkPack->AnyHaveMetadataValue('TargetFramework', '$(LatestTargetFramework)')))" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <!--
+      Reference base shared framework at incoming dependency flow version, not bundled sdk version.
+      Apply this to all projects that target the default tfm (e.g. net8.0) or a rid-based variant of it (e.g. net8.0-windows)
+    -->
+    <FrameworkReference
```

Comment:
Duplicate this for Microsoft.AspNetCore.App w/ the incoming aspnetcore dependency version